### PR TITLE
Updated Notification Badge

### PR DESCRIPTION
**Pull request by marmat**
I changed the looks of the notification badge a bit so that it looks like the original Google+ notification badge (i.e. rounded corners and a stroke around them, see images for comparison). Furthermore I made the badge being rendered as inactive whenever the count is a non-positive number (if this is not desired, make sure to set `newItem` to false when calling the method with a `count` of zero, I didn't look where the method gets called).

Please see the comments in my commit; maybe the method for drawing a rounded rectangle should be added to the Canvas context prototype somewhere else.

Active Badges

Inactive Badges

### CODE REVIEW

File: js/background_controller.js [hunk 1]
```diff
@@ -121,17 +121,68 @@ BackgroundController.prototype.onMessageListener = function(request, sender, sen
...
+    // http://goo.gl/DFnaA)
+    fillAndStrokeRoundRect(ctx, 0.5, 0.5, 18, 18, 2);
   }
-  ctx.fillRect(0, 0, 19, 19);
+
   ctx.font = 'bold 11px arial, sans-serif';
-  ctx.fillStyle = '#fff';
+  ctx.fillStyle = newItem ? '#fff' : '#999';
 
-  chrome.browserAction.setTitle({title: count + ' hangouts are going on right now!'});
```
Comment (marmat) [inline js/background_controller.js:134]: Sorry, another error from copying and pasting... please revert this change when merging

File: js/background_controller.js [hunk 2]
```diff
@@ -139,10 +190,7 @@ BackgroundController.prototype.drawBadgeIcon = function(count, newItem) {
     ctx.fillText(count + '', 3, 14);
   }
   else if (count >= 0) {
-    ctx.fillText(count + '', 6, 14);
-    if ( count == 0 ){
-      chrome.browserAction.setTitle({title: 'There are no hangouts going on!'});
-    }
```
Comment (marmat) [inline js/background_controller.js:145]: See comment above :-/

**Comment by marmat** [comment]
I moved the methods for drawing rounded rectangles into a separate file. Now the CanvasRenderingContext2D prototype is extended by these methods and therefore the methods are available everywhere.

Furthermore I reverted the accidental changes which I've commented on above.

**Comment by ghost** [comment]
@mohamedmansour, how can I pull this into a branch other than master? (leaving it for now)

**Comment by mohamedmansour** [comment]
Awesome work @kaktus621 !  =)
